### PR TITLE
Feature/audio rtpc filters

### PR DESCRIPTION
**Pull request by Dartemiss**
What's new?
- Methods for setting the RTPC.

### CODE REVIEW

File: Engine/Module/ModuleAudio.cpp [hunk 1]
```diff
@@ -114,4 +114,14 @@ void ModuleAudio::SelectMainListener()
 			return;
 		}
 	}
-}
\ No newline at end of file
+}
+
+void ModuleAudio::SetRTPCValue(const std::string & name, int value)
```
Comment (mariofv) [inline Engine/Module/ModuleAudio.cpp]: Code Style

File: Engine/Module/ModuleAudio.cpp [hunk 2]
```diff
@@ -114,4 +114,14 @@ void ModuleAudio::SelectMainListener()
...
+{
+	AK::SoundEngine::SetRTPCValue(name.c_str(), value);
+}
+
+void ModuleAudio::ResetRTPCValues(const std::string & name)
```
Comment (mariofv) [inline Engine/Module/ModuleAudio.cpp]: Code Style

File: Engine/Module/ModuleTime.cpp [hunk 3]
```diff
@@ -116,11 +117,13 @@ void ModuleTime::Play()
 		SetTimeScale(1.f);
 		frame_start_time = game_time_clock->Read();
 		App->animations->PlayAnimations();
+		App->audio->ResetRTPCValues("FilterLPF");
```
Comment (mariofv) [inline Engine/Module/ModuleTime.cpp]: Better lets call this in a script, here we will be mixing game logic with engine

**Review by mariofv — APPROVED** [review]
Just change the things in the comments, rest looks good!

**Review by RickyVimon — APPROVED** [review]

**Pull request merged**